packages/ses/src/lockdown-shim.js

```diff
@@ -131,6 +131,7 @@ export const repairIntrinsics = (options = {}) => {
     overrideTaming = getenv('LOCKDOWN_OVERRIDE_TAMING', 'moderate'),
     stackFiltering = getenv('LOCKDOWN_STACK_FILTERING', 'concise'),
     domainTaming = getenv('LOCKDOWN_DOMAIN_TAMING', 'safe'),
+    evalTaming = getenv('LOCKDOWN_COMPARTMENT', 'safeEval'),
```

Comment:
Was this intentional? I expected
```suggestion
    evalTaming = getenv('LOCKDOWN_EVAL_TAMING', 'safeEval'),
```